ROADMAP.md

```diff
@@ -1,10 +1,58 @@
 # Kubeflow Roadmap
 
 ## Kubeflow 1.5 Release, Due: February 2022
-* Kubeflow 1.5 timeline is tracked in [#538](https://github.com/kubeflow/community/pull/538)
-* Additional details will be posted in early 1Q'22
+* Kubeflow 1.5 [milestones and timeline](https://github.com/kubeflow/community/pull/538)
+
+### Themes
+* Increase ML pipeline functionality with aggregation and passing of parameters and artifacts between pipeline components
+* Improve model accuracy and reduce overfitting, especially with hyper parameter tuning
+* Simplify operations and optimize utilization (including spot instance use cases for distributed training) 
+* More consistent user experience - UI appearance, features and naming
+* Improved documentation, tutorials and examples
+* Stretch - Support for K8s 1.22 and associated dependencies (cert mgr, istio)
+
+### Major Features from each Working Group (note: Individual WG versions are independent of Kubeflow's)
+
+#### Kubeflow Pipelines, v1.8
+* [Emissary executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor) is default rather than Docker runtime
```

Comment:
Switching to [Emissary executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor) enables Kubeflow Pipelines deployment on Kubernetes >= v1.20, which runs on containerd runtime instead of Docker runtime.